<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SEKDPB4" failures="3" tests="7" name="ChromeTest" time="0.111" errors="0" timestamp="2023-02-09T10:17:54 IST">
  <testcase classname="testNgPack.Automation" name="test1" time="0.013"/>
  <testcase classname="testNgPack.Automation" name="test2" time="0.003">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	URL is wrong expected [https://www.facebook.com/home] but found [https://www.facebook.com/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
URL is wrong expected [https://www.facebook.com/home] but found [https://www.facebook.com/]
at testNgPack.Automation.test2(Automation.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="testNgPack.Automation" name="test3" time="0.001"/>
  <testcase classname="testNgPack.TestNGClass" name="testA" time="0.003">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at testNgPack.TestNGClass.testA(TestNGClass.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testA -->
  <testcase classname="testNgPack.TestNGClass" name="testB" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNgPack.TestNGClass.testB(TestNGClass.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testB -->
  <testcase classname="testNgPack.TestNGClass" name="testC" time="0.002"/>
  <testcase classname="testNgPack.TestNGClass" name="testD" time="0.003"/>
</testsuite> <!-- ChromeTest -->
